CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v2.0.0] - Monday, April 18th, 2022
```

Comment:
let's update the README to reflect all these changes.

Reply:
it looks like there are still a couple `wt.track("event", params)` references

Reply:
whoops, should be fixed now, thanks for catching